### DISCUSSION TITLE
Only repository name showing on the public link created by Github

### DISCUSSION THREAD

**Discussion by PARTHS333-cmd** [community/community#29463] [Other Feature Feedback, Questions, & Ideas]
🙋‍♂️ Hey there everyone I'm up with a problem, for first time I am making a website, using html. I checked my codes I it seems to be all right, but after I created the the page link it only shows the repository name that's "CV".
I have no idea what do now🙁. Any help will be heartily❤ appreciated.
<img width="584" alt="Screenshot 2022-08-13 022023" src="https://user-images.githubusercontent.com/66141169/184571225-0878f366-b66e-4485-a0ed-9ef0c4ab28b8.png">

**Comment by radar-blip** [comment]
For some reason, your github pages is pointing to your readme.md and not your index.html
I'm not sure why though, I cant figure it out yet. 

Your contact page is working fine here: https://parths333-cmd.github.io/CV/contact.html
but not your index.html

**Reply by PARTHS333-cmd** [reply]
thanks for your effort man..😊 I really appreciate it.

**Reply by radar-blip** [reply]
No problem. I tried, just couldn't pinpoint the issue.
Anyway, have a good day!

**Reply by FerdausAhmedNasim** [reply]
I am also having the same problem

**Comment by surjendu104** [comment]
I have fixed that issue and your site is running correctly. Check the pull request in your repo.
here's the url --- [PARTH SARTHI](https://surjendu104.github.io/CV/)

ps merge the pull request

**Reply by Arv18** [reply]
Interesting

**Comment by airtower-luna** [comment]
You have a really subtle error: The file that *should* be `index.html` is actually called `<SPACE>index.html`. :sweat_smile: You can see that [in the build log](https://github.com/PARTHS333-cmd/CV/runs/7813291485?check_suite_focus=true#step:5:24). Fix the filename and it should work.


You can also disable the default Jekyll processing (which is what turns the README into HTML) by adding a `.nojekyll` file to your repository, see: https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#static-site-generators

**Reply by surjendu104** [reply]
yeah! noticed that.....

**Reply by PARTHS333-cmd** [reply]
yup..😅 thanks bro❤..

**Reply by PARTHS333-cmd** [reply]
thank you very much @airtower-luna 😊🤍

**Reply by radar-blip** [reply]
Also, check spelling in Title of index.html 👍🏼

**Reply by InspireEmber** [reply]
Thankyou very much!!

**Comment by GitnamanHub** [comment]
In my site also only repository name (CV) is showing. Can anyone help me with it?

**Reply by surjendu104** [reply]
May be you have deployed the 'docs' page in GitHub pages.
To check this 
1. Goto your repository
2. Click the settings tab
3. Click on the page section
4. Under 'Branch' check you have selected the '/root'
5. Then deploy again
![Screenshot_2023-02-27-11-36-23-85](https://user-images.githubusercontent.com/101238933/221487480-f7bf57f5-c074-40d3-a9e3-3b2d734591f0.jpg)

**Reply by airtower-luna** [reply]
If you mean https://github.com/GitnamanHub/CV (make sure to include a link in the future, it's much easier to help that way!), the problem is that you don't have an `index.html` in your repository. Because it's missing, the README file gets turned into one. Name the file that you want to see when opening your site `index.html`.

**Reply by GitnamanHub** [reply]
I have changed the name but still it's not showing

**Reply by GitnamanHub** [reply]
Thanks after trying again it is done but can you explain why it needs file name to be index

**Reply by airtower-luna** [reply]
It's an ancient convention (at least since 1993) for web servers to show `index.html` (if it exists) when the user accesses a directory. Some web servers allow configuring a different name, but that's rarely used, and you can't do it on GitHub pages.

**Comment by GitnamanHub** [comment]
Thanks for replying but I have selected the root option only!

**Comment by GitnamanHub** [comment]
So will I have to always name my main file I as index,
Also is it only in html or in any other language also I have to do this.
BTW thanks for replying!

**Reply by airtower-luna** [reply]
What do you mean by "other language"? Websites have to be in HTML. If you use another programming language to generate the HTML it depends on the language/tool and how you run it.

**Reply by mubarakali826** [reply]
airtower-luna plz help me my issue is nt resolved any of method discussed by you

**Comment by GitnamanHub** [comment]
Thank you for the clarification.

**Comment by Zherrymi** [comment]
I just have the same issue and I don't know what are my mistakes

**Reply by GitnamanHub** [reply]
Try making your main file name to index.html
It helped me.

**Comment by Tuesday69** [comment]
I have the same issue,  just only my respiratory name showed up

**Reply by surjendu104** [reply]
Rename main html file as index.html

**Reply by PRAFULLADHAKE** [reply]
i am new . I have the same issue, just only my respiratory name showed up in my react app
please help me

**Comment by mubarakali826** [comment]
Ok I struggled with this issue alot and now I am here with multiple solutions,try any one of them  and your issue will be resolved.

1) check that your index.html file is included in repository.

2) check your index.html name and spellings (it should be without space.

3)Check that your branch is set to root in settings.

4) don't keep your files in a folder,keep them freely on github like

Index.html
Style.css
Index.js
Or any other files
Not inside any folder.

My issue was resolved with 4th one😁

**Reply by Darshana343** [reply]
Hey,Thanks a lot! your 4th suggestion worked..

P.S. Note that the image files(.png) can be placed inside the folders but not html's

**Reply by FerdausAhmedNasim** [reply]
thanks for solution

**Comment by JatinTaiwala** [comment]
hey can anyone please help me i'm also having the same issue. only my repo name is displaced on my website page.
my url is: https://jatintaiwala.github.io/JatinPortfolio.github.io/

**Reply by GitnamanHub** [reply]
Change index.html.html to index.html

**Comment by 123-Jawad** [comment]
Hey everyone I'm up with the problem i am making website using react.js and node.js but I created the the page link it only shows the repository name
![Screenshot 2023-07-07 002147 (2)](https://github.com/community/community/assets/121505838/408fb117-4601-4a32-8f3e-b32384d867ad)

**Comment by surjendu104** [comment]
Github pages support hosting static pages
However to host react apps you have to use some npm packages and little bit of configuration. make sure you have done that. 
for reference you can check these sites
1. [SIte-1](https://blog.logrocket.com/deploying-react-apps-github-pages/#:~:text=The%20simplicity%20of%20deploying%20a,own%20custom%20domain%20or%20subdomain.)
2. [Site-2](https://www.freecodecamp.org/news/deploy-a-react-app-to-github-pages/)

**Comment by Kirandev242144** [comment]
it worked for me after i unpublish the site and created static.yml by generating static html (it created github folder with static.yml

**Comment by Papi2199** [comment]
hello does anyone know how to take off the repository name from my project? https://papi2199.github.io/perlamiabimba/ idk if is a silly question haha I'm new to this and I was only trying to do something for my girlfriend haha thank for reading I hope to find someone to help me
![Screenshot 2023-12-30 023536](https://github.com/community/community/assets/155129664/4425e31d-3055-46c6-8667-890b9a5da57a)
 :)

**Comment by InspireEmber** [comment]
Yes the previous issue has been solved. But I have a new issue in which a
html file link is not opening, it's showing error 404, I have the linked
file in my public folder and in the index folder I have given a link to it

**Comment by InspireEmber** [comment]
Yes I would like to understand it

**Comment by InspireEmber** [comment]
A static website generator for my uploaded html files

**Comment by InspireEmber** [comment]
Yes it is, every other link is working but only one is showing error 404
